trigger-actions-framework/main/default/classes/TriggerBaseTest.cls

```diff
@@ -238,5 +283,11 @@ private class TriggerBaseTest {
 		}
 		public void afterUndelete(List<SObject> newList) {
 		}
+		public Map<Id, Integer> getIdToNumberOfTimesSeenBeforeUpdate() {
+			return TriggerBase.idToNumberOfTimesSeenBeforeUpdate;
+		}
+		public Map<Id, Integer> getIdToNumberOfTimesSeenAfterUpdate() {
+			return TriggerBase.idToNumberOfTimesSeenAfterUpdate;
+		}
```

Comment:
Using `base.idToNumberOfTimesSeenBeforeUpdate()` does not work since static methods cannot be referenced from a non static context.
I removed the additional getters in BaseTest, now accessing by using `TriggerBase.idToNumberOfTimesSeenBeforeUpdate.get(MY_ACCOUNT.Id)`.